executors/kubeexec/config.go

```diff
@@ -28,6 +28,9 @@ type KubeConfig struct {
 	User      string `json:"user"`
 	Password  string `json:"password"`
 	Namespace string `json:"namespace"`
+	// Use Secrets to get the Bearerkey
+	UseSecrets bool   `json:"use_secrets"`
+	Secret     string `json:"secret"`
```

Comment:
This is looking really good.  The only real suggestion I have is to change `Secret` to `TokenFile`, just like we have `CertFile`